pkgs/development/python-modules/werkzeug/default.nix

```diff
@@ -15,17 +16,21 @@
 
 buildPythonPackage rec {
   pname = "werkzeug";
-  version = "2.2.3";
-  format = "setuptools";
+  version = "2.3.6";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
```

Comment:
This version of werkzeug dropped support for Python 3.7.

Reply:
fixed in [force-push](https://github.com/NixOS/nixpkgs/compare/77adbeb69e1da83bc8f87e05667345fbbc00aa77..c17fc82201157b2f00e5520052ac8de30062f78c)